src/test/java/com/epam/edumanagementsystem/security/config/AuthenticationTest.java

```diff
@@ -23,14 +23,14 @@ public class AuthenticationTest {
     @Autowired
     private MockMvc mockMvc;
 
-    @Test
-    public void testLoggingWithValidUser() throws Exception {
-        mockMvc.perform(formLogin()
-                .user(rightEmail).password(password))
-                .andExpect(redirectedUrl("admins"))
-                .andExpect(status().isFound())
-                .andExpect(authenticated());
-    }
+//    @Test
```

Comment:
Do not comment unit tests remove or specify with annotation that it should not be run